contrib/pkginstaller/Makefile

```diff
@@ -8,8 +8,10 @@ else
 endif
 GVPROXY_VERSION ?= 0.7.3
 VFKIT_VERSION ?= 0.5.1
+KRUNKIT_VERSION ?= 0.1.0
 GVPROXY_RELEASE_URL ?= https://github.com/containers/gvisor-tap-vsock/releases/download/v$(GVPROXY_VERSION)/gvproxy-darwin
 VFKIT_RELEASE_URL ?= https://github.com/crc-org/vfkit/releases/download/v$(VFKIT_VERSION)/vfkit-unsigned
+KRUNKIT_RELEASE_URL ?= https://github.com/containers/krunkit/releases/download/v0.1.1/krunkit-podman-unsigned-0.1.1.tgz
```

Comment:
Shouldn't this use KRUNKIT_VERSION rather then 0.1.1 And this version is 0.1.1 versus 0.1.0 above?

Reply:
Yes, I've put this draft in a haste and didn't pay much attention. Should be better now.